packages/kubernetes-1.23/kubernetes-1.23.spec

```
@@ -39,6 +39,7 @@ Source11: kubelet-server-crt
 Source12: kubelet-server-key
 Source13: etc-kubernetes-pki-private.mount
 Source14: credential-provider-config-yaml
+Source15: logdog.kubernetes-1.23.conf
```

Let's keep the file name the same across packages. That way it's easier to compare the directories via `git diff --no-index`:
```suggestion
Source15: logdog.kubelet.conf
```